### DISCUSSION TITLE
How to log server-side fetch calls?

### DISCUSSION THREAD

**Discussion by gavrilikhin-d** [vercel/next.js#71010] [App Router]
```
  logging: {
    fetches: {
      fullUrl: true
    }
  }
```
Doesn't seem to add logs for server-side fetches. Only client-side

**Comment by SidKH** [comment]
Hey Daniil 👋

What Next.js version do you use? 
Here is an example showing server-side fetches on Next.js v. 14.2.14

https://github.com/user-attachments/assets/465e7c52-5147-4ef2-a9bb-4aa4c6f7230c

**Reply by gavrilikhin-d** [reply]
Hey, I'm using latest nextjs (15.0.1). 
Thanks for you response!
I see some of the requests now, but not all. Could it be because of static routes or sever actions inside client components?

**Reply by gavrilikhin-d** [reply]
I would show me only `POST <route>` log and no logs for actual server action
